client/src/components/App/Content/InvestigationForm/TabManagement/ClinicalDetails/useClinicalDetails.tsx

```diff
@@ -26,16 +32,60 @@ const useClinicalDetails = (parameters: useClinicalDetailsIncome): useClinicalDe
         );
     };
 
+    const getStreetByCity = (cityId: string) => {
+        axios.get('/addressDetails/city/' + cityId + '/streets').then(
+            result => result && result.data && setStreetsInCity(result.data.map((node: Street) => node))
+        )};
+    
+    const getClinicalDetailsByEpidemiologyNumber = () => {
+        axios.post('/clinicalDetails/getInvestigatedPatientClinicalDetailsFields?epidemiologyNumber=' + epidemiologyNumber).then(
+            result => {
+                if (result && result.data && result.data.data && result.data.data.investigationByEpidemiologyNumber) {
```

Comment:
Our standart is to do as much data parsing as you can on the server (like removing nodes and big fields names